ocs_ci/ocs/fiojob.py

```diff
@@ -339,6 +342,13 @@ def workload_fio_storageutilization(
             fio is stored on the volume, and reclaim policy of the volume is
             changed to ``Retain`` so that the volume is not removed during test
             teardown for later verification runs
+        delete_fio_data_after_test (bool): If true, delete fio data after the fio
+            storageutilization is completed. Else if False, delete the fio data at
+            the end of all the tests. If if false you also need to provide the
+            request parameter.
+        request: The built-in pytest request object.
```

Comment:
I would write that this is required if `delete_fio_data_after_test` is set to false.